src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -510,9 +524,8 @@ private String resolveContent(String result, Set<String> usedIds) {
             return Optional.of(EntryUtil.capitalizeFirst(entry.getType()));
         }
 
-        // TODO: Changed this to also consider alias fields, which is the expected
+        // Changed this to also consider alias fields, which is the expected
```

Comment:
I have no idea when it was introduced, so hard to say. @JabRef/developers keep comment or remove (since we haven't found any drawbacks)?